config/locales/idv/en.yml

```diff
@@ -31,8 +31,8 @@ en:
       select_financial_account: Select a financial account…
       ssn_label_html: Social Security Number %{tooltip}
       state: State
-      use_ccn: Provide credit card number
-      use_financial_account: Provide financial account number
+      use_ccn: Provide credit card number >
```

Comment:
As much as I like fancy glyphs, we're already using angle brackets elsewhere (like cancel and back links). Let's stick to the existing design pattern and flag as something to review in a design audit.